static/src/javascripts/bootstraps/enhanced/accessibility.js

```diff
@@ -82,13 +84,13 @@ define([
 
         React.render(
             React.createElement(Accessibility),
-            document.getElementById(module.DOM_ID),
+            document.getElementById(DOM_ID),
```

Comment:
this change (and in main.js) has the tiny side effect of making `module.DOM_ID` read only. It's good that tests don't change that value but maybe you can juggle `var module` around in order to keep the same behaviour